keyboards/sofle/original/keymaps/default/keymap.c

```diff
@@ -0,0 +1,394 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file is (essentially) identical to the keyhive default.

It would make more sense to move the entire keymap folder back (so it's "keyboards/sofle/keymaps").  This way, it's shared between the different versions.

Reply:
There are different variants of the sofle keyboard available. Some of them have per key RGB, some of them underglow RGB or both. Some keymaps make use of different RBG features, some may not. I don't think it would make sense to try to generalize these keymaps into a shared folder. Or is there a way to tag a keymap to a certain variant?